pytest_django/fixtures.py

```diff
@@ -323,3 +327,21 @@ def _live_server_helper(request):
     """
     if 'live_server' in request.funcargnames:
         request.getfuncargvalue('transactional_db')
+
+
+@pytest.fixture(scope='function')
+def assert_num_queries(db, pytestconfig):
```

Comment:
I see now that it is nice to use the `db` fixture automatically - but it should also work when using `transactional_db` in the test (which we should handle internally already).
So, just add a test where you use the `transactional_db` fixture explicitly.